README.md

```diff
@@ -1524,13 +1508,23 @@ func (e Email) BuildJSONSchema() (map[string]interface{}, error) {
 }
 ```
 
+### Sub-schema Limitation
+
+Sub-schemas only get's converted to JSON Schema, if you specify a sub-schema via setting a Field's `Validator` attribute to a `schema.Object` instance. Use of the Field's `Schema` field is _not_ supported. Instead we hope #77 will be implemented.
+
 ### schema.Dict Limitations
 
 `schema.Dict` only support `nil` and `schema.String` as `KeysValidator` values. Note that some less common combinations of `schema.String` attributes will lead to usage of an `allOf` construct with duplicated schemas for values. This is to avoid usage of regular expression expansions that only a subset of implementations actually support.
 
 The limitation in `KeysValidator` values arise because JSON Schema draft 4 (and draft 5) support for key validation is limited to [properties, patternProperties and additionalProperties](https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.4.4). This essentially means that there can be no JSON Schema object supplied for key validation, but that we need to rely on exact match (properties), regular expressions (patternProperties) or no key validation (additionalProperties).
 
-## schema.URL Limitations
+### schema.Reference Provisional Support
+
+The support for `schema.Reference` is purely provisional, and simply returns an empty object `{}`, meaning it does not give any hint as to which validation the server might use.
+
+With a potential later implantation of the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) (a.k.a. the Swagger 2.0 Specification), the goal is to refer to the ID field of the linked resource via an object `{"$ref": "#/definitions/<unique schema title>/id"}`. This is tracked via issue #36.
```

Comment:
I thought issue references should work as per [GFM](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown). Maybe it only applies to comments and commit messages?

I could add the link syntax just to be sure.